static/js/main.js

```diff
@@ -27,6 +27,16 @@
 
     $(function(){
         $(document)
+        .on('click', '.contribute-header', function(){
+            var el = $(this);
+            if(el.hasClass('shown')){
+                el.removeClass('shown');
+                el.next().slideUp({duration: 500, easing: 'easeInOutQuart'});
+            } else {
+                el.addClass('shown');
+                el.next().slideDown({duration: 500, easing: 'easeInOutQuart'});
+            }
+        })
```

Comment:
Do y'all still need this js as well?

Reply:
Still evaluating that and the css. I just wanted to get the content changes pushed up.

Reply:
Would appear that the css and this js snippet (or their removal) have no effect on the page.